runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -246,9 +247,11 @@ def createRunnerV2ValidatesRunnerTest = { Map args ->
 // task directly ('dependsOn buildAndPushDockerContainer'). This ensures the correct
 // task ordering such that the registry doesn't get cleaned up prior to task completion.
 task buildAndPushDockerContainer() {
-  dependsOn ":sdks:java:container:docker"
+  def javaVer = project.hasProperty('compileAndRunTestsWithJava11') ? "java11" : "java8"
+  dependsOn ":sdks:java:container:${javaVer}:docker"
+  finalizedBy 'cleanUpDockerImages'
```

Comment:
The `finalizeBy` definition here breaks the Kafka Performance Test[1] because it makes the docker image get cleaned up as soon as it is pushed.

We define the cleanup logic here: https://github.com/apache/beam/blame/master/runners/google-cloud-dataflow-java/build.gradle#L280-L287

[1] https://ci-beam.apache.org/job/beam_PerformanceTests_Kafka_IO/